src/coreclr/jit/gentree.cpp

```diff
@@ -848,6 +848,28 @@ bool GenTree::gtHasReg(Compiler* comp) const
             }
         }
     }
+    else if (IsCopyOrReloadOfMultiRegHWIntrinsic())
+    {
+        const GenTreeCopyOrReload* copyOrReload = AsCopyOrReload();
+        const unsigned             regCount     = copyOrReload->gtGetOp1()->GetMultiRegCount(comp);
+        // A Multi-reg copy or reload node is said to have regs,
+        // if it has valid regs in any of the positions.
+
+        for (unsigned i = 0; i < regCount; ++i)
+        {
+            hasReg = (copyOrReload->GetRegNumByIdx(i) != REG_NA);
+            if (hasReg)
+            {
+                break;
+            }
+        }
+    }
+    // else if (IsMultiRegHWIntrinsic())
+    //{
+    //    // For intrinsic that returns multiple values, we always assign
+    //    // them registers to all or none and hence checking if the reg
+    //    // is assigned at 0th index using GetRegNum() is sufficient.
+    //}
```

Comment:
Delete?

Reply:
That was intentional for future reference.